tests/test_tasks.py

```diff
@@ -417,6 +417,47 @@ class MyTask(Task):
         mytask = MyTask()
         execute(mytask)
 
+    @server(port=2200)
+    @server(port=2201)
+    def test_nested_execution_with_explicit_ports(self):
+        """
+        nested executions should work with defined ports
+        """
+
+        def sub_task_one():
```

Comment:
Matter of personal taste I guess, but I think the main task would read more clearly if this was named something like `assert_port_is_2201`